src/components/App.tsx

```diff
@@ -24,20 +35,58 @@ const theme = createMuiTheme({
       '"Segoe UI Emoji"',
       '"Segoe UI Symbol"',
     ].join(','),
+    h1: {
+      fontSize: 56,
+      lineHeight: 80,
+    },
+    h2: {
+      fontSize: 48,
+      lineHeight: 72,
+    },
+    h4: {
+      fontSize: 24,
+      lineHeight: 36,
+    },
+    body1: {
+      fontSize: 18,
+      lineHeight: 30,
+    },
+    body2: {
+      fontSize: 36,
+      lineHeight: 50,
+    },
```

Comment:
These values don't make sense and break things. `body1` and `body2` are used for normal text and `body2` should be smaller than `body1` (see the [Material-UI docs](https://material-ui.com/components/typography/#component)).